src/config/services/managing.md

```diff
@@ -60,14 +60,14 @@ create a file named `down` in its service directory:
 To disable a service, remove the symlink from the running runsvdir:
 
 ```
-# rm /var/service/<service>
+# unlink /var/service/<service>
```

Comment:
Why?

Reply:
> This is actually something I wanted other's opinions on. I prefer using unlink because the action is more obvious and I think for documentation purposes it's more explicit.

I can revert to `rm`, though.

Reply:
`rm` is more commonly used and well known, "unlink" doesn't mean remove a link, both tools use the unlink syscall.

Reply:
`unlink` is the same as `rm`, its not exclusively for removing {sym,}links and there is no reason to really prefer unlink here over rm, most users know rm and its shorter to write.